contracts/src/systems/hyperstructure/initialize_hyperstructure.cairo

```diff
@@ -0,0 +1,26 @@
+#[system]
+mod InitializeHyperStructure {
+    use eternum::alias::ID;
+    use eternum::components::hyperstructure::HyperStructure;
+
+    use dojo::world::Context;
+
+    fn execute(ctx: Context, entity_id:ID, hyperstructure_id: ID) {   
```

Comment:
my initial thought was that initializing the hyperstructure would cost resources as well, what do you think @ponderingdemocritus

Reply:
same comment as above